app/controllers/search_controller.rb

```diff
@@ -69,15 +70,10 @@ def sort_by(sort_by_param)
   end
 
   def wrap_instances
-    @ids_to_results = @searchdata.results.map { |r| [r._source[:id], r] }.to_h
-    instances = self.class::SEARCHED_MODEL.where(id: @ids_to_results.keys)
-    # Because the instances are in db order, we'll need to re-sort them to
-    # preserve the elasticsearch order. This is preferable to iterating through
-    # the elasticsearch results and fetching one db item each time, however;
-    # we used to do that and all those roundtrips were costing us in
-    # performance.
+    # #records fetches the database instances while maintaining the search
```

Comment:
`#records` => `records`?

Reply:
Actually intended to be `#records`, as it's referring to a method.